src/web-experiments.ts

```diff
@@ -151,7 +151,7 @@ export class WebExperiments {
                 `/api/web_experiments/?token=${this.instance.config.token}`
             ),
             method: 'GET',
-            transport: 'XHR',
+            transport: 'fetch',
```

Comment:
can this just not specify their own transport and use the default instead?

didn't realise there were three when i started commenting on each 🤣